sickbeard/helpers.py

```diff
@@ -114,65 +114,81 @@ def remove_non_release_groups(name, clean_proper=False):
     # Check your database for funky release_names and add them here, to improve failed handling, archiving, and history.
     # select release_name from tv_episodes WHERE LENGTH(release_name);
     # [eSc], [SSG], [GWC] are valid release groups for non-anime
+
+    # All lowercase and alphabetically sorted
+    groups = {
+        '-20-40',
+        '1044',
+        'agusiq',
+        'androidtwou',
+        'brassetv',
+        'bt',
+        'buymore',
+        'chamele0n$',
+        'cttv',
+        'english',
+        'ethd',
+        'ettv',
+        'eztv',
+        'german',
+        'giuseppetnt',
+        'glodls',
+        'gz',
+        'ipt',
+        'italian',
+        'ksks',
+        'monster',
+        'movietam',
+        'mp4remuxer',
+        'musicbolt.com',
+        'nlsub',
+        'no.rar',
+        'nzbgeek',
+        'obfuscated',
+        'oldeztv',
+        'p2pdl',
+        'pawulon',
+        'publichd',
+        'rarbg',
+        'rartv',
+        'renc',
+        'rick',
+        'ripsalot',
+        'rp',
+        'seedbox',
+        'siklopentan',
+        'silv4',
+        'simple@blurg',
+        'sparr',
+        'sparrow',
+        'spastikustv',
+        'talamasca32',
+        'tjet',
+        'utr',
+        'vtv',
+        'www.cpasbien.com',
+        'www.cpasbien.pe',
+        'www.cpasbien.pw',
+        'www.newpct1.com',
+        'www.scenetime.com',
+        'www.tensiontorrent.com',
+        'www.torrentday.com',
+        'www.usabit.com',
+        'youtubetomp3.uno'
+    }
+
+    non_groups_re = r'(({delimiter}+)({groups})({delimiter}*)$)|(^({delimiter}*)({groups})({delimiter}+))'.format(
```

Comment:
Here's an example with the generated regex: https://regex101.com/r/vY9sM0/2

The idea is:
- `delimiter` 1 or more times + non group name + `delimiter 0 or more times` + `end`
- `start` + `delimiter` 0 or more times + non group name + `delimiter 1 or more times`